packages/block-library/src/cover/edit.js

```diff
@@ -47,6 +47,7 @@ import {
 	backgroundImageStyles,
 	dimRatioToClass,
 } from './shared';
+import { attributesFromMedia } from './utils';
```

Comment:
Sorry, I know I suggested `utils` but then I realized we already have `shared.js`, which might be a better place to put that function as well

Reply:
No problem! I saw that file but the name wasn't that clear to me, I'll move it to the existent `shared` file

Reply:
Updated https://github.com/WordPress/gutenberg/pull/19722/commits/a61cbb64fa7f1952b245b088818fbc5b0bfa3ac6#diff-73868aa369f61d82259cb4dd4cc7130b